modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -120,6 +120,15 @@ const ATTRIBUTE_MAPS = {
   }
 };
 
+const ATTRIBUTE_TRANSITION = {
+  enter: (value, chunk) => {
+    if (chunk.length) {
+      return chunk.subarray(chunk.length - value.length);
```

Comment:
The parameter names `value` and `chunk` do not seem immediately suggestive, but perhaps I just haven't thought through what they do.

Also, seems like a lot of small object allocations. No big concern for now, but maybe take another look at this as we build out "chunk" index handling for partial updates.